scvi/dataloaders/_anntorchdataset.py

```diff
@@ -67,7 +67,7 @@ def setup_getitem(self):
         >>> sd = AnnTorchDataset(adata_manager)
 
         # following will only return the X and batch both by default as np.float32
-        >>> sd.setup_getitem(getitem_tensors  = ['X,'batch'])
+        >>> sd.setup_getitem(getitem_tensors  = ['X', 'batch'])
```

Comment:
Am I correct thinking that this is an old API?  Looks like`.setup_getitem` method only accept the `self` argument.

Reply:
That seems correct, `getitem_tensors` should be given to the class constructor